tests/test_helper/bats/cluster.bats

```diff
@@ -46,3 +46,115 @@ setup() {
+        expected_addr+=$(test_is_ipv6_test && echo "]" || true)
+        local expected_port
+        [ "$nbsb" == "nb" ] && expected_port=6643 || expected_port=6644
+
+        run microovn_ovndb_cluster_status "$container" "$nbsb"
+
+        assert_success
+        assert_line "Address: ssl:${expected_addr}:${expected_port}"
+        assert_line "Status: cluster member"
+    done
+}
+
+@test "OVN Northbound DB clustered" {
+    _test_db_clustered nb
+}
+
+@test "OVN Southbound DB clustered" {
+    _test_db_clustered sb
+}
+
+function _get_member_cluster_address() {
+    # build a list of cluster addresses by interrogating every member
+    #
+    # Note that we do it this way intentionally to help uncover any
+    # inconsistencies (as opposed to asking a single member for all the
+    # addresses).
+    for container in $TEST_CONTAINERS; do
+        container_services=$(microovn_get_cluster_services "$container")
+        if [[ "$container_services" == *"central"* ]]; then
+            echo "$(microovn_get_cluster_address $container)"
+        fi
+    done
+}
+
+@test "Chassis Open_vSwitch external_ids:ovn-remote addresses" {
+    local cluster_addresses=()
+    local container_services
+
+    readarray -t cluster_addresses < <(_get_member_cluster_address)
+    assert_equal "${#cluster_addresses[@]}" 3
+    for container in $TEST_CONTAINERS; do
+        container_services=$(microovn_get_cluster_services "$container")
+        if [[ "$container_services" != *"chassis"* ]]; then
+            echo "Skip $container, no chassis services" >&3
+            continue
+        fi
+
+        run lxc_exec \
+            "$container" \
+            "microovn.ovs-vsctl get Open_vSwitch . external_ids:ovn-remote"
+        for addr in "${cluster_addresses[@]}"; do
+            expected_addr=$(test_is_ipv6_test && echo "[" || true)
+            expected_addr+=$addr
+            expected_addr+=$(test_is_ipv6_test && echo "]" || true)
+            assert_output -p "$expected_addr"
+        done
+    done
+}
+
+function _test_db_connection_string() {
+    local nbsb=$1; shift
+
+    local check_var
+    [ "$nbsb" == "nb" ] && \
+        check_var=OVN_NB_CONNECT || \
+        check_var=OVN_SB_CONNECT
+    local expected_port
+    [ "$nbsb" == "nb" ] && \
+        expected_port=6641 || \
+        expected_port=6642
+
+    local cluster_addresses=()
+
+    _get_member_cluster_address
+    readarray -t cluster_addresses < <(_get_member_cluster_address)
+    assert_equal "${#cluster_addresses[@]}" 3
+    for container in $TEST_CONTAINERS; do
+        run lxc_exec \
+            "$container" \
+            "grep ^$check_var /var/snap/microovn/common/data/ovn.env"
+        for addr in "${cluster_addresses[@]}"; do
+            expected_addr=$(test_is_ipv6_test && echo "[" || true)
+            expected_addr+=$addr
+            expected_addr+=$(test_is_ipv6_test && echo "]" || true)
+            assert_output -p "ssl:${expected_addr}:${expected_port}"
```

Comment:
Similar concern about the partial matching, although the accidental match is probably less likely in this case

Reply:
The string here has equal format to the `external_ids:ovn-remote`, so if we fix that we can re-use the parsing code here.

Reply:
As commented above, looking at this again the extra effort/looping to get exact matches here is not worth it, the fully qualified search string should be good enough to avoid false matches.